src/main/java/nz/co/jammehcow/jenkinsdiscord/DiscordPipelineStep.java

```diff
@@ -0,0 +1,123 @@
+package nz.co.jammehcow.jenkinsdiscord;
+
+import hudson.Extension;
+import hudson.model.TaskListener;
+import nz.co.jammehcow.jenkinsdiscord.exception.WebhookException;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution;
+import org.jenkinsci.plugins.workflow.steps.StepContextParameter;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+
+public class DiscordPipelineStep extends AbstractStepImpl {
+
```

Comment:
It's nothing major, but removing that newline under the class' curly brace would be nice. Fits with the rest of the classes.